tests/tests/market_tests.cpp

```diff
@@ -406,9 +410,11 @@ BOOST_AUTO_TEST_CASE(hardfork_core_338_test)
  */
 BOOST_AUTO_TEST_CASE(hardfork_core_453_test)
 { try {
-   auto mi = db.get_global_properties().parameters.maintenance_interval;
-   generate_blocks(HARDFORK_CORE_453_TIME - mi); // assume all hard forks occur at same time
-   generate_blocks(db.get_dynamic_global_properties().next_maintenance_time);
+
+   if(hf1270)
+      generate_blocks(HARDFORK_CORE_1270_TIME);
+   else
+      generate_blocks(HARDFORK_CORE_343_TIME);
```

Comment:
To be more accurate, here we should generate blocks until the maintenance time but not the hf time.